packages/liveblocks-core/src/protocol/BaseActivitiesData.ts

```diff
@@ -0,0 +1,5 @@
+import type { ActivityData } from "./InboxNotifications";
+
+export type BaseActivitiesData = {
+  [key: `$${string}`]: ActivityData;
```

Comment:
For the same reason as with our other base types, we should use
```suggestion
  [key: `$${string}`]: ActivityData | undefined;
```
here I think.

Reply:
I'm not sure because it's an intermediate type: it doesn't represent any real data (unlike `ActivityData`) and it's never used by users, only augmented. It's only a way for users to give us a mapping "a kind" ↔ "its activity data type", and I don't think we want them to be allowed to give us `undefined` for a kind.

Reply:
Oh, I see. That's actually a good point. Discard my comment 👍